### PR TITLE
Adding community info links

### PR DESCRIPTION
**Pull request by matzew**
Infos on how to connect with the community for this NPM module

### CODE REVIEW

**Comment by fh-automatron** [comment]
### Pull Request Quality Guidelines

As we continue to focus on quality, here are a few things to keep in mind:

* The code changes conform to the contributing & code style guides for this component (if they exist)
* A component owner should review your change (we've tagged these above)
* Automated testing has been added or updated for this change (e.g. UART, Unit/Acceptance tests)
* If updating server component this PR should be deployed to a grid somewhere (via `fhcap-cli`) - please provide a link to this
* Ensure JIRA is up to date before & after merge (e.g. deployment notes, tester notes, release notes, fix version)
* Any SonarQube issues should be addressed where applicable (use your own judgement)

Please ping a component owner (see `fh contrib`) when your builds are complete, changes are deployed and you're ready for a review.
Fhcap PR will be generated automatically.
To deploy your changes to a cluster place comment in this PR with pattern [x][y] where x=deploy and y=clustername you wish to deploy. 
You can also deploy your change manually using fhcap-cli

File: README.md [hunk 1]
```diff
@@ -10,6 +10,15 @@ fh-amqp-js
...
 FeedHenry AMQP client wrapper for communication with a Rabbit cluster.
 
+|                 | Project Info  |
+| --------------- | ------------- |
+| License:        | Apache License, Version 2.0  |
+| Build:          | npm  |
+| Documentation:  | http://docs.feedhenry.com/v3/api/cloud_api.html  |
+| Issue tracker:  | https://issues.jboss.org/projects/FH/summary  |
+| Mailing list:  | [feedhenry-dev](https://www.redhat.com/archives/feedhenry-dev/) ([subscribe](https://www.redhat.com/mailman/listinfo/feedhenry-dev))  |
+
```
Comment (sebastienblanc) [inline README.md]: We could even add the IRC channel #feedhenry , wdyt ?

**Comment by matzew** [comment]
@sebastienblanc good point! I've added Info on IRC - note `irc://` links are not supported by GH markdown, therefore I just used text ...

https://github.com/jch/html-pipeline/blob/5feb0c185fb1e5037e3ad6d6bd533fb4af3685d8/lib/html/pipeline/sanitization_filter.rb#L40

File: README.md [hunk 2]
```diff
@@ -10,6 +10,16 @@ fh-amqp-js
...
 FeedHenry AMQP client wrapper for communication with a Rabbit cluster.
 
+|                 | Project Info  |
+| --------------- | ------------- |
+| License:        | Apache License, Version 2.0  |
+| Build:          | npm  |
+| Documentation:  | http://docs.feedhenry.com/v3/api/cloud_api.html  |
+| Issue tracker:  | https://issues.jboss.org/projects/FH/summary  |
+| Mailing list:   | [feedhenry-dev](https://www.redhat.com/archives/feedhenry-dev/) ([subscribe](https://www.redhat.com/mailman/listinfo/feedhenry-dev))  |
+| IRC:            | #feedhenry channel in the [freenode](http://freenode.net/) network.  |
```
Comment (grdryn) [inline README.md]: Do you think it would be useful to link the `#feedhenry` part to https://webchat.freenode.net/?channels=feedhenry?

Comment (matzew) [inline README.md]: @grdryn ah, good point!

updated!

**Comment by odra** [comment]
:+1:

**Pull request merged**